.storybook/preview.jsx

```diff
@@ -29,11 +33,21 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  msw: {
+    handlers: [
+      // Only return mocked data when making requests to our mock stack
+      ...getHandlers(MOCK_REPO_ORIGIN),
+    ],
+  },
 }
 
-import { StorybookComponentWrapper } from '../src/lib/containers/StorybookComponentWrapper'
+// Initialize MSW
+initialize({
+  onUnhandledRequest: 'bypass',
```

Comment:
MSW will still be active when making requests to real stacks, so we don't want it to report warnings or throw errors.